### ISSUE TITLE
error in android smartphone after using scrcpy

### ISSUE THREAD

**Issue by suhailiealx** [Genymobile/scrcpy#4377]
- [*] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [*] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 
 - scrcpy version: 1.12.1
 - installation method: Windows release
 - device model: Android
 - Android version: 8.1

Hello, i am new to the scrcpy. I use scrcpy to try monitoring the smartphone screen from a pc using usb cable. After i disconnect the connection, there is error in my android that always give message "due to corrupted desktop data, it has been restored to the default layout.". My android screen become unresponsive and stuck at home screen without displaying any apps.

does anyone has any ideas for this error?

**Comment by rom1v** [comment]
Nope, and I would be surprised it could be related to scrcpy in any way.
